### DISCUSSION TITLE
How to remove an item by drag outside

### DISCUSSION THREAD

**Discussion by galr52** [react-grid-layout/react-grid-layout#1797] [General]
Hey

I used this example (https://react-grid-layout.github.io/react-grid-layout/examples/15-drag-from-outside.html) to insert new items to the layout by drag from outside.

I am looking for an example how to remove an item from the layout by drag the item outside of the RGL boundary 

React-Grid-Layout [v1.3.4]

any help?

**Comment by Felixhuangsiling** [comment]
Hi did you manage to fix the issue?

**Reply by galr52** [reply]
I'm not exactly sure, but I think I found a solution that worked. I ended up changing how item removal works, so I no longer needed the drag-out behavior.
